src/sbd-inquisitor.c

```diff
@@ -39,6 +39,26 @@ bool sync_resource_startup = false;
 
 int parse_device_line(const char *line);
 
+static int
+sanitize_numeric_option_value(const char *value)
+{
+    char *end = NULL;
+    int result = -1;
+
+    if (value == NULL) {
+        return -1;
+    }
+
+    result = strtol(value, &end, 10);
```

Comment:
guess you have to set errno = 0 for strtol.

Reply:
Aah could that be the reason for test failures.